### PR TITLE
Alex autoupdate relevant

### PR DESCRIPTION
**Pull request by ghost**
make the relevant right now update without having to refresh page

- created custom hook that auto checks every 5s, sets state => rerender when date range starts/ends
- i have zero ide awhat the idfference between activeTime and displayTime are

https://user-images.githubusercontent.com/22948432/107864756-84786b00-6e2d-11eb-8a54-dc6eacc55e0d.mov

### CODE REVIEW

**Comment by anikarabenhorst** [comment]
active time is when should we should be able to see the card, display is for the text that is on the card (like an event time or the time a form needs to be completed by).

**Comment by ghost** [comment]
gotcha, thanks! I think I did it correctly then

File: src/hooks/useShowTime.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  useEffect(() => {
+    console.log("update!");
+    const interval = setInterval(() => {
+      // every 5 seconds, check whether or not the status has changed.
+      // if so, then set state
+      const newShouldShow = isTimeBetween(startTime, endTime);
+      if (newShouldShow !== shouldShow) {
+        setShouldShow(newShouldShow);
+      }
```
Comment (nupur-n) [inline src/hooks/useShowTime.js:17]: just to double check, this doesn't do anything with the API call right? I just want to make sure we aren't continually calling to air table. It doesn't look like it but

Comment (ghost) [reply src/hooks/useShowTime.js:17]: yep! basically all this hook does is return true or false depending on whether the interval includes the current time

the useAirTableAPI hook is completely untouched, as well as the entire `RelevantRightNow` component! this is just for the individual cards

Comment (ghost) [reply src/hooks/useShowTime.js:17]: oh yes I should clarify it only sets state when the previous result was different – ie, when we go from shouldShow = false => shoudShow = true, or vice versa

**Comment by ghost** [comment]
Whoops, accidentally rebased and merged and undid my own commits. Everything should be good now, with no reformatting stuff

**Comment by ghost** [comment]
closed in favor of #48

**Pull request closed without merging**